module/filter/basic_kalman.cpp

```diff
@@ -2,4 +2,55 @@
 
 namespace basic_kalman {
 
+RM_kalmanfilter::RM_kalmanfilter() : KF_(4, 2) {
```

Comment:
```suggestion
KalmanFilter::KalmanFilter() : KF_(4, 2) {
```

类命名规范，`KF_()` 继承参数未说明